### ISSUE TITLE
Documentation Lacking on how to use the token received to make other API requests.

### ISSUE THREAD

**Issue by piyush0609** [Tivix/django-rest-auth#351]
The Documentation doesn't explain how to use the token obtained on login or registration. How to configure the API view to for proper execution of  API calls.

**Comment by maxim-kht** [comment]
Hi @piyush0609, the view configuration depends on which Token model you use. For example, by default django-rest-auth uses default Django REST Framework Token: http://www.django-rest-framework.org/api-guide/authentication/#tokenauthentication.

**Comment by piyush0609** [comment]
@maxim-kht Yeah understood. Maybe this is fault on the DRF's part as in their documentation it is not mentioned how to use authentication_class = (TokenAuthentication) and permission_class. I faced this issue while using IsAuthenticated in permission_class along with authentication_class set as Token Authentication. But thankfully now I have figured it out.
I was just making sure that an example should be included django-rest-auth about how to use the token for user authorization while making API requests to other endpoints(apart from rest_auth.urls).
